wasm/test_page/bergamot.html

```diff
@@ -127,6 +127,7 @@
 cpu-threads: 0
 quiet: true
 quiet-translation: true
+gemm-precision: int8shift
```

Comment:
For maximum performance this should be `int8shiftAlphaAll` or `int8shiftAlpha`  (not sure which of the two since I haven't tested inside the WASM environment) https://github.com/browsermt/students/blob/3344c53660acc160714ff8394c8155db65cf9c13/eten/enet.student.tiny11/config.intgemm8bitalpha.yml#L20